### ISSUE TITLE
standardize parameter broken

### ISSUE THREAD

**Issue by habernal** [jitsi/asr-wer#3]
MWE:

```
from jiwer import wer

e1 = wer("he's my neminis", "he is my <unk> [laughter]", standardize=True)
e2 = wer("he is my neminis", "he is my")

assert e1 == e2
```
According to the documentation, these should be equivalent, but are not.

Version 1.3 from PyPi

**Comment by nikvaessen** [comment]
Hi, 

Thanks for your interest in this repo! I've created an initial fix at PR #4 . Do you have some other concerns before I merge?

**Comment by habernal** [comment]
@nikvaessen Thanks for the fix! No other concerns at the moment.
